### ISSUE TITLE
win32_interop.lib and ws2_32.lib conflict

### ISSUE THREAD

**Issue by etertal** [microsoftarchive/redis#577]
Redis relies on win32_interop.lib and activeMQ dependent ws2_32.lib library conflicts, but I need these two libraries. How do you solve this conflict?. Running shows that many functions are redefined

**Comment by shezio** [comment]
You might wanna try this one (am new to redis but this looks promising 🙏)

https://www.google.co.il/url?sa=t&source=web&rct=j&url=https://stackoverflow.com/questions/23824286/visual-studio-c-redis-msopentech-port-client-hiredis&ved=0ahUKEwiRp-K5leHXAhVJuhQKHZlxB_0QFggnMAE&usg=AOvVaw0jNUizZhjDDjUxjHOhpWav

**Comment by isaf** [comment]
I have the same problem.
Did you find the answer?

**Comment by etertal** [comment]
No,I used redis version is 2.6 ,It has a hiredis library that doesn't 
 need win32_interop.lib. url : https://github.com/MicrosoftArchive/redis/tree/2.6.
